server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -638,6 +638,43 @@ public void run() {
    */
   private Map<DataLevel,Set<KeyExtent>> partitionMigrations() {
     final Map<DataLevel,Set<KeyExtent>> partitionedMigrations = new EnumMap<>(DataLevel.class);
+
+    // Don't try to check migrations if the Root and Metadata
+    // tables are not hosted.
+    boolean skipMigrationCheck = false;
+
+    if (watchers.size() != 3) {
+      log.debug("Skipping migration check, not all TabletGroupWatchers are started");
+      skipMigrationCheck = true;
+    } else {
+      for (TabletGroupWatcher watcher : watchers) {
+        if (!watcher.isAlive()) {
+          log.debug("Skipping migration check, not all TabletGroupWatchers are started");
+          skipMigrationCheck = true;
+          break;
+        }
+        final DataLevel level = watcher.getLevel();
+        if (level == DataLevel.ROOT || level == DataLevel.METADATA) {
+          final TableId tid = level == DataLevel.ROOT ? SystemTables.ROOT.tableId()
+              : SystemTables.METADATA.tableId();
```

Comment:
This use of `DataLevel` doesn't seem correct. See:
https://github.com/apache/accumulo/blob/f676217e757d823aff0499ff5af1beda2abc5f13/core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java#L82-L85